linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/bash/shared.sh

```diff
@@ -5,4 +5,21 @@
 
 populate var_system_crypto_policy
 
-update-crypto-policies --set ${var_system_crypto_policy}
+stderr_of_unsuccessful_call=$(update-crypto-policies -h 2>&1 > /dev/null)
+
+if test "$?" = 0; then
+	stderr_of_unsuccessful_call=$(update-crypto-policies --set ${var_system_crypto_policy} 2>&1 > /dev/null)
```

Comment:
Is there a reason why not to make this call directly?

Reply:
The problem with error messages is that e.g. `Error: Unknown policy ...` doesn't make it obvious what program could have said it, so I captured the error, and used it in a more explanatory error message.

Reply:
Optionally, you can replace the previous occurrence of `update-crypto-policies` by `which`.

Reply:
I'm sorry I was not clear.
What I originally meant was: In line 8, why not do the actual set policy command there?
And condition if on line 17 can be based value of `$?`?

Reply:
Interestingly, `which` may not be available (check out e.g. the Fedora container). Anyway, I gave it a shot, assuming that RC of 127 means command not found.